server/data/assessmentStore.ts

```diff
@@ -1,16 +1,17 @@
-import type { RedisClient } from './redisClient'
+import { hoursToSeconds } from 'date-fns'
 
+import type { RedisClient } from './redisClient'
 import logger from '../../logger'
 import { AssessmentPage, SubmittedInput } from './model/immediateNeedsReport'
-import { secondsUntilMidnight } from '../utils/utils'
 
-const assessmentPrefix = 'assessment:'
-const answeredQuestionsPrefix = 'answered:'
-const currentPagePrefix = 'currentPage:'
-const editedQuestionPrefix = 'edit:'
+const defaultTimeToLive = hoursToSeconds(24 * 5)
```

Comment:
you could make this configurable in case Nick wants to test it in DEV with a shorter span

Reply:
That's a good idea, thanks